apps/ui/src/helpers/validation.ts

```diff
@@ -200,6 +215,8 @@ function getErrorMessage(errorObject: Partial<ErrorObject>): string {
         return 'Must be an integer.';
       case 'ethValue':
         return 'Must be a number.';
+      case 'addresses-with-voting-power':
+        return 'Must be a valid address with voting power.';
```

Comment:
```suggestion
        return 'Must be a valid list of addresses with voting power.';
```

To make it clear that you can have multiple.